Formula/vis.rb

```diff
@@ -38,12 +38,16 @@ def install
       system "luarocks", "build", "lpeg", "--tree=#{luapath}"
     end
 
-    system "./configure", "--prefix=#{libexec}"
+    system "./configure", "--prefix=#{prefix}"
     system "make", "install"
+
+    # Rename vis & the matching manpage to avoid clashing with the system.
+    man1.install man1/"vis.1" => "vise.1"
+    (libexec/"bin").install bin/"vis"
```

Comment:
maybe use env_script_all_files if we're having to export anyway and then `mv` vis?